src/components/Footer/index.jsx

```diff
@@ -0,0 +1,30 @@
+import { Favorite, GitHub, LinkedIn, Twitter } from "@material-ui/icons";
+import React from "react";
+import "./styles.css";
+
+const Footer = () => {
+    return (
+        <div className='footer'>
+            <div className='text'>
+                <p>
+                    Made with{" "}
+                    <Favorite style={{ color: "red", padding: "0px 2px" }} /> by
```

Comment:
Avoid inline styling

Reply:
yeah It's an icon and the external styles sheet takes extra margin don't know why I will take care of these thank you for PR